tutorial/debezium-cassandra-init/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM cassandra:3.11
+
+RUN apt-get update && \
+    apt-get install -y wget curl sudo vim nano git gnupg
+
+RUN sudo apt install -y openjdk-8-jdk
+
+ENV DEBEZIUM_VERSION=1.4.0.Alpha2 \
+    MAVEN_CENTRAL="https://repo1.maven.org/maven2" \
+    CASSANDRA_YAML=/opt/cassandra/conf \
+    DEBEZIUM_HOME=/debezium
+
+COPY cassandra.yaml $CASSANDRA_YAML
+
+RUN mkdir $DEBEZIUM_HOME
+RUN curl -fSL -o $DEBEZIUM_HOME/debezium-connector-cassandra.jar \
+                 $MAVEN_CENTRAL/io/debezium/debezium-connector-cassandra/$DEBEZIUM_VERSION/debezium-connector-cassandra-$DEBEZIUM_VERSION-jar-with-dependencies.jar
+
+COPY log4j.properties $DEBEZIUM_HOME
```

Comment:
I've just checked the Docker file docs and it seems that the `COPY` instuctions can be merged into a single one as the destination dir is same for all files